site/openapiSchema.yaml

```diff
@@ -0,0 +1,307 @@
+openapi: 3.0.4
+info:
+  title: Transfers API - 1.1 (Draft)
+  version: 1.0.0
+
+paths:
+  /v1/transfers/search:
+    get:
+      summary: Search for available transfers
+      description: Returns a list of available transfers based on pick-up and drop-off details
+      parameters:
+        Please refer to the parameters section in the existing documentation for details
+      responses:
+        Please refer to the responses section in the existing documentation for details
+
+  /v1/transfers/{productCode}:
+    post:
+      summary: Book a transfer
+      description: Create a booking for a given transfer product
+      parameters:
+        requestBody:
+          description:
+          required: true
+          content:
+            application/x-www-form-urlencoded:
+              schema:
+                $ref: Please refer to the parameters section in the existing documentation for details
+      responses:
+        Please refer to the responses section in the existing documentation for details
+
+  /v1/transfers/booking/{bookingRef}:
+    get:
+      summary: Retrieve booking details (future)
+      description: Get full information about an existing booking
+      parameters:
+        - name: bookingRef
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Booking details successfully retrieved.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  Booking:
+                    type: object
+                    properties:
+                      BookingRef:
+                        type: string
+                        example: "BookingRef"
+                      SupplierRef:
+                        type: string
+                        example: "SupplierReference"
+                      VoucherRef:
+                        type: string
+                      BookingStatus:
+                        type: string
+                        example: "PENDING"
```

Comment:
I guess it's worth updating the status here too now...

Reply:
I suppose it is now up to us to decide what we want to return here.

Reply:
I have changed it to lower case for now